rest/src/main/java/org/jboss/pnc/rest/trigger/BuildTriggerer.java

```diff
@@ -178,39 +181,22 @@ private void signalBpmEvent(String uri) {
                 log.error("Error parsing BPM config.", e);
             }
         }
-        
-        SSLContextBuilder builder = new SSLContextBuilder();
-        try {
-            builder.loadTrustMaterial(null, new TrustStrategy() {
-                @Override
-                public boolean isTrusted(X509Certificate[] chain, String authType) throws CertificateException {
-                    return true;
-                }
-            });
-        } catch (NoSuchAlgorithmException | KeyStoreException e1) {
-            e1.printStackTrace();
-        }
-
-        SSLConnectionSocketFactory sslSF = null;
-        try {
-            sslSF = new SSLConnectionSocketFactory(builder.build(), SSLConnectionSocketFactory.ALLOW_ALL_HOSTNAME_VERIFIER);
-        } catch (KeyManagementException | NoSuchAlgorithmException e1) {
-            e1.printStackTrace();
-        }
 
-        CloseableHttpClient httpclient = HttpClients.custom().setSSLSocketFactory(sslSF)
-                .setHostnameVerifier(new AllowAllHostnameVerifier()).build();
+        /*
+         * TODO: Do not ignore certificates, rather setup servers properly.
+         */
+        CloseableHttpClient httpClient = HttpUtils.getPermissiveHttpClient();
 
         HttpPost request = new HttpPost(uri);
         request.addHeader("Authorization", getAuthHeader());
         log.info("Executing request " + request.getRequestLine());
 
-        try (CloseableHttpResponse response = httpclient.execute(request)) {
+        try (CloseableHttpResponse response = httpClient.execute(request)) {
             log.info(response.getStatusLine());
             response.close();
-            httpclient.close();
+            httpClient.close();
```

Comment:
Yeah I think so... BTW it looks pretty awkward that a response needs calling close...